src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
+import machines.Bulldozer;
+import machines.Excavator;
+import machines.Machine;
+import machines.Truck;
+
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = {new Excavator(), new Bulldozer(), new Truck()};
 
+        for (Machine machine: machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
+    }
```

Comment:
Consider using abstract references instead of specific ones where possible to allow for polymorphism. For example, instead of creating an array of `Machine` directly, you could use a `List<Machine>` which would allow for more flexibility if you decide to add more machines or change the collection type later on.